src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the `loading` state and the `useDMMessage` hook seems appropriate for managing direct message functionality. However, error handling within the `sendMessageDM` function should be reviewed to ensure that any errors are properly handled and communicated to the user.



<!-- This is an auto-generated comment by OSS CodeRabbit -->